Docs note on Cross-account observability permissions for CW datasource (#95034)
grafana/grafana: docs/sources/datasources/aws-cloudwatch/_index.md

}
```

> {{% admonition type="note" %}}
> Using CloudWatch Cross-Account querying functionality allows you to retrieve metrics and logs across different regions. However, it does not allow querying EC2 Instance Attributes across regions. The EC2 API doesn't support querying across different accounts because this ia a CloudWatch characteristic, not an EC2 one.
> {{% /admonition %}}

Reviewer: Here's a suggestion that simplifies some of the language.

I've left two placeholders in:

- _`<THIS>`_: What does this refer to in the context of the admonition? Perhaps we should be explicit.
- _`<CHARACTERISTIC>`_: I want to check this is the preferred word. More descriptive and less complicated words like 'feature' might be better. Also the I saw reference to "instance attributes" and I wanted to make sure that characteristic didn't mean "instance attribute" here.

Finally, the admonition shortcode syntax with `{{<` and `>}}` is preferred even though I know the rest of the file uses `{{%` and `%}}`. I'll submit a PR to patch all of those.

```suggestion
{{< admonition type="note" >}}
Using CloudWatch Cross-Account querying functionality lets you to retrieve metrics and logs across different regions but it doesn't let you query EC2 Instance Attributes across regions.

The EC2 API doesn't support querying across different accounts because <THIS> is a CloudWatch <CHARACTERISTIC>, not an EC2 one.
{{< /admonition >}}
```

Author: Thank you for the check @jdbaldry, modified, let me know if not is more understandable

}
```

> {{< admonition type="note" >}}
> Using CloudWatch Cross-Account querying functionality lets you to retrieve metrics and logs across different regions but it doesn't let you query EC2 Instance Attributes across regions.
> The EC2 API does not support querying across different accounts because cross-account observabilityis a CloudWatch feature, not an EC2 one.
> {{< /admonition >}}

Reviewer: OK, so from my understanding of the documentation in [Cross-account observability](https://grafana.com/docs/grafana/latest/datasources/aws-cloudwatch/query-editor/#cross-account-observability) it's not about cross-region but cross-account querying in a single region.

```suggestion
{{< admonition type="note" >}}
Cross-account observability lets you to retrieve metrics and logs across different accounts in a single region but you can't query EC2 Instance Attributes across accounts because those come from the EC2 API and not the CloudWatch API.
{{< /admonition >}}
```
